comp/networkpath/npcollector/npcollectorimpl/npcollector_test.go

```diff
@@ -211,7 +211,7 @@ func Test_NpCollector_runningAndProcessing(t *testing.T) {
 	}
 	npCollector.ScheduleConns(conns)
 
-	waitForProcessedPathtests(npCollector, 5*time.Second, 1)
+	waitForProcessedPathtests(npCollector, 5*time.Second, 2)
```

Comment:
Few lines below, we are expecting 2 pathtest to be processed:

```
assert.Equal(t, uint64(2), npCollector.processedTracerouteCount.Load())
```

Hence, we need to wait for 2 pathtest before continuing.